integration/nevermined/NFT721Templates.test.ts

```diff
@@ -190,42 +193,38 @@ describe('NFT721Templates E2E', () => {
                 collector2.getId()
             )
 
-            const payload = decodeJwt(config.marketplaceAuthToken)
-            const metadata = getMetadata()
-            metadata.userId = payload.sub
-
-            royaltyAttributes = getRoyaltyAttributes(
-                nevermined,
-                RoyaltyKind.Standard,
-                royalties
-            )
-
-            const nftAttributes = NFTAttributes.getInstance({
-                metadata,
-                price: assetPrice1,
-                serviceTypes: ['nft-sales', 'nft-access'],
-                ercType: 721,
-                nftType: NeverminedNFT721Type.nft721,
-                nftContractAddress: nft.address,
-                preMint: false,
-                nftTransfer: false,
-                royaltyAttributes: getRoyaltyAttributes(
-                    nevermined,
-                    RoyaltyKind.Standard,
-                    0
-                )
-            })
-            ddo = await nevermined.nfts721.create(nftAttributes, artist)
         })
 
         describe('As an artist I want to register a new artwork', () => {
-            it('I want to register a new artwork and tokenize (via NFT). I want to get 10% royalties', async () => {
-                await nft.nftContract.grantOperatorRole(artist.getId(), nftContractOwner)
+            it('I want to register a new artwork and tokenize (via NFT). I want to get 10% royalties', async () => {                
 
-                await nft.mint(ddo.shortId(), artist)
+                const payload = decodeJwt(config.marketplaceAuthToken)
+                const metadata = getMetadata()
+                metadata.userId = payload.sub    
 
-                const balance = await nft.balanceOf(artist)
-                assert.deepEqual(balance, BigNumber.from(1))
+                royaltyAttributes = getRoyaltyAttributes(
+                    nevermined,
+                    RoyaltyKind.Standard,
+                    royalties
+                )
+    
+                const nftAttributes = NFTAttributes.getInstance({
+                    metadata,
+                    price: assetPrice1,
+                    serviceTypes: ['nft-sales', 'nft-access'],
+                    ercType: 721,
+                    nftType: NeverminedNFT721Type.nft721,
+                    nftContractAddress: nft.address,
+                    preMint: true,
+                    nftTransfer: false,
+                    royaltyAttributes: getRoyaltyAttributes(
+                        nevermined,
+                        RoyaltyKind.Standard,
+                        0
```

Comment:
supposedly the test says that wants to get 10% of the royalties but here is defined as 0